src/content/docs/en/reference/experimental-flags/fonts.mdx

```diff
@@ -325,6 +325,34 @@ import { Font } from 'astro:assets';
 <Font cssVariable="--font-roboto" preload />
 ```
 
+If not specified or set to `false`, no preload links will be emitted. If set to `true`, all possible links will be included.
+
+#### Granular preloads
+
+<p>
+<Since v="5.15.0" />
+</p>
+
+If you need more control over which font file is preloaded, you can instead provide an array of objects:
+
+The following example will only preload font files with a `400` weight or a `normal` style in the `latin` subset:
+
+```astro title="src/components/Head.astro" {7-10}
+---
+import { Font } from 'astro:assets';
+---
+
+<Font
+  cssVariable="--font-roboto"
+  preload={[
+    { subset: 'latin', style: 'normal' },
+    { weight: '400' },
+  ]}
+/>
+```
+
+If a candidate is a file with a variable weight, it will be preloaded if the passed weight is within range. For example, a font file for font weight `100 900` will be included since `400` is within that range.
```

Comment:
I don't know what the word "candidate" means here (I sort of do, but it feels like it's a special term or jargon when it's not, so we should explain in more explicit terms.)

Do you mean something like:

```suggestion
Variable weight font files will be preloaded if any weight within its range is requested. For example, a font file for font weight `100 900` will be included when `400` is specified in a `preload` object.
```

But also, will ALL the weights 100 to 900 be dowloaded, or just the 400 in this case? I would want to specify that, too.

Reply:
> But also, will ALL the weights 100 to 900 be dowloaded

A variable font file contains all weights for the given range so yeah they would all be downloaded, which is expected.